db/migrate/20220219192029_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.date :CreatedAt
+      t.date :UpdatedAt
```

Comment:
- [ ] Here, `CreatedAt` and `UpdatedAt` are redundant. Kindly remove them because the columns will automatically be added with `t.timestamps`
- [ ] Kindly make the above change in all the other tables.